src/main/native/cpp/jni/CameraServerJNI.cpp

```diff
@@ -197,26 +195,21 @@ static jobject MakeJObject(JNIEnv *env, const cs::VideoMode &videoMode) {
       static_cast<jint>(videoMode.fps));
 }
 
-static jobject MakeJObject(JNIEnv *env, const cs::RawEvent &event) {
+static jobject MakeJObject(JNIEnv* env, const cs::RawEvent& event) {
   static jmethodID constructor =
       env->GetMethodID(videoEventCls, "<init>",
                        "(IIILjava/lang/String;IIIIIIILjava/lang/String;)V");
   JLocal<jstring> name(env, MakeJString(env, event.name));
   JLocal<jstring> valueStr(env, MakeJString(env, event.valueStr));
   return env->NewObject(
-      videoEventCls,
-      constructor,
-      static_cast<jint>(event.kind),
+      videoEventCls, constructor, static_cast<jint>(event.kind),
```

Comment:
I don't like this.. with how many arguments this has it really needs to be one line per arg.